cmd/acra_configui/acra_configui.go

```diff
@@ -105,12 +116,37 @@ func SubmitSettings(w http.ResponseWriter, r *http.Request) {
 	w.Write(jsonToServer)
 }
 
+func parseTemplate() {
+	tplPath := fmt.Sprintf(filepath.Join(*staticPath, "index.html"))
```

Comment:
what about to use function parameter `staticPath` (`func parseTemplate(staticPath string)`) instead global variable?

Reply:
done